packages/docs/docusaurus.config.js

```diff
@@ -21,7 +21,7 @@ const config = {
   organizationName: 'fixie-ai', // Usually your GitHub org/user name.
   projectName: 'ai-jsx', // Usually your repo name.
 
-  onBrokenLinks: 'throw',
+  onBrokenLinks: 'warn',
```

Comment:
Why are we changing this?

Reply:
I'm having some weird problems building with this branch and believe something must have broken with link resolution, so this works around that temporarily while I figure out what's going on.